### PR TITLE
REST-ful dashboard and insights generation.

### PR DESCRIPTION
**Pull request by dkorolev**
REST-browsable sessions, data export, insights generation math.

TODO(dkorolev):
1. Retire ticks, as we correctly have them coming from the server now.
2. Visualize insights.

@mzhurovich : Please take a look.

Thanks,
Dima

### CODE REVIEW

**Comment by dkorolev** [comment]
Diffstats before data files removal: `+134,448 −9`
Diffstats after data files removal: `+1,428 −9`

**Comment by dkorolev** [comment]
And after removing old code: `+1,315 −9`

File: src/insights.h [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    std::string key;
+    // Binary features set for this session. `FeatureInfo` is present for each of them.
+    std::vector<std::string> feature;
+    template <typename A>
+    void serialize(A& ar) {
+      ar(CEREAL_NVP(key), CEREAL_NVP(feature));
+    }
+  };
+  // Realm defines the universe within which sessions can be analyzed as the whole.
+  // such that comparing them to each and detecting anomalies is what yields insights.
```
Comment (mzhurovich) [inline src/insights.h:65]: s/`whole.`/`whole,`/
s/`to each`/`with each other`/

Comment (dkorolev) [inline src/insights.h:65]: :+1: 

Fixed in my branch.

**Comment by mzhurovich** [comment]
@dkorolev Let's merge it!

**Pull request merged**